pkg/devfile/parser/parse.go

```diff
@@ -431,6 +454,24 @@ func parseFromRegistry(importReference v1.ImportReference, resolveCtx *resolutio
 	return DevfileObj{}, fmt.Errorf("failed to get id: %s from registry URLs provided", id)
 }
 
+func getStackFromRegistry(id, registryURL, stackDir, destDir string) error {
+	if !strings.HasPrefix(registryURL, "http://") && !strings.HasPrefix(registryURL, "https://") {
+		return fmt.Errorf("the provided registryURL: %s is not a valid URL", registryURL)
+	}
+	registryLibrary.PullStackFromRegistry(registryURL, id, stackDir, registryLibrary.RegistryOptions{})
```

Comment:
when version is specified, stack should be `<stack>:<version>`, if no version specified, use `<stack>`